doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2228,7 +2228,7 @@ existential variable used in the same context as its context of definition is wr
    Check (fun x y => _) 0 1.
 
 Existential variables can be named by the user upon creation using
-the syntax ``?``\ `ident`. This is useful when the existential
+the syntax ``?``\ [`ident`]. This is useful when the existential
```

Comment:
Use ```:n:`?[@ident]` ``` instead.

Reply:
I don't know what that means but ok, I'll just copy and paste :)